content/intro-to-storybook/react/en/test.md

```diff
@@ -43,16 +43,14 @@ git checkout -b change-task-background
 
 Change `src/components/Task.js` to the following:
 
-```js
-// src/components/Task.js
-
+```diff:title=src/components/Task.js
 <div className="title">
   <input
     type="text"
     value={title}
     readOnly={true}
     placeholder="Input title"
-    style={{ background: 'red' }}
++   style={{ background: 'red' }}
```

Comment:
Why is there a `+` here?

Reply:
It's a diff, it means that the plus and green line are additions

Reply:
Word I see now